tools/bin/hawqregister

```diff
@@ -1145,7 +1153,7 @@ def main(options, args):
 
     failure_handler = FailureHandler(conn)
     # register
-    if options.yml_config and ispartition(options.yml_config):
+    if check_file_exist(options.yml_config) and ispartition(options.yml_config):
```

Comment:
I just think check_file_exist() should not be called in the if statement. In this statement, it may be confused the condition for whether is partition or not.